src/commands/configuration/togglepii.ts

```diff
@@ -1,14 +1,29 @@
 import { DB } from '@root/config';
 import { SageUser } from '@lib/types/SageUser';
-import { Message } from 'discord.js';
+import { CommandInteraction, Message } from 'discord.js';
 import { DatabaseError } from '@lib/types/errors';
 import { Command } from '@lib/types/Command';
 
 export default class extends Command {
 
-	description = `Toggles whether or not your personally identifiable information will be sent by instructors over Discord.`;
+	description = `Toggles whether your personally identifiable information will be sent to instructors over Discord.`;
 	aliases = ['pii'];
 
+	async tempRun(interaction: CommandInteraction): Promise<void> {
+		const entry: SageUser = await interaction.client.mongo.collection(DB.USERS).findOne({ discordId: interaction.user.id });
+
+		if (!entry) {
+			throw new DatabaseError(`Member ${interaction.user.username} (${interaction.user.id}) not in database`);
```

Comment:
Does this return an interaction/message reply or just a log message (or related)? It would be worth making this an interaction reply if the former is not the case